### ISSUE TITLE
Remembering manually downloaded mods(or all)

### ISSUE THREAD

**Issue by Explodey-Wolf** [PolyMC/PolyMC#801]
### Role

Downloading modpacks, with OPTED OUT MODS

### Suggestion

I want PolyMC to CACHE the opted out mods, so you don't need to download them again.

### Benefit

So I don't need to download them again.

### This suggestion is unique

- [X] I have searched the issue tracker and did not find an issue describing my suggestion, especially not one that has been rejected.

### You may use the editor below to elaborate further.

PolyMC could also cache normal mods(preferably a setting to make this happen) just for when cf is being mean. Thanks!

**Comment by Explodey-Wolf** [comment]
Also, make it so you can delete some of the mods. Don't want this 24gb mod cached? Begone! Also blacklists.

**Comment by Explodey-Wolf** [comment]
Oops :)

**Comment by luni3359** [comment]
Blacklists how?

**Comment by Explodey-Wolf** [comment]
> Blacklists how?

Well, I would assume you would somehow select a mod, and add it to a list. It would not automatically cache them if that's the case. 

But, another idea is to have a selection box next to each opted-out mod(in the list) which would only cache them. This would work with scanning for the mods in your downloads folder, and caching from there. This could also lead to making it automatically add those mods to the modpack.

**Comment by DioEgizio** [comment]
#339 basically

**Comment by Explodey-Wolf** [comment]
Not really, as this only skips the downloading opted out mod, but that issue _does_ sound pretty cool
